### ISSUE TITLE
Bug: Unexpected enableTrace Parameter Injection in BedrockFlowsAgent Calls

### ISSUE THREAD

**Issue by guisartori88** [awslabs/multi-agent-orchestrator#163]
### Expected Behaviour

When invoking the BedrockFlowsAgent in a Lambda environment, setting enableTrace=False (or not specifying it at all) should result in no enableTrace parameter being sent to Bedrock. If enableTrace=True is supported as per documentation, it should also not cause the request to fail due to unknown parameters.

### Current Behaviour

In AWS Lambda, requests to Bedrock Flows fail with the following error, whether enableTrace is not specified or explicitly set to True:

`Parameter validation failed:
Unknown parameter in input: "enableTrace", must be one of: flowAliasIdentifier, flowIdentifier, inputs
`

This issue does not occur when running the same code locally. It appears that enableTrace is being injected internally or not recognized by Bedrock, causing the Bedrock API to reject the payload.

### Code snippet

```python
from multi_agent_orchestrator.agents import BedrockFlowsAgent, BedrockFlowsAgentOptions

def my_flow_input_encoder(agent, input, **kwargs):
    # Minimal input encoder
    return {"inputs": {"file_key": input["inputs"]["file_key"]}}

# Tried both with and without enableTrace:
# - Without enableTrace parameter
# - With enableTrace=True as per documentation

sentiment_agent = BedrockFlowsAgent(BedrockFlowsAgentOptions(
    name="sentiment-agent",
    description="Agent for sentiment analysis",
    flowIdentifier="yy",
    flowAliasIdentifier="yy",
    flow_input_encoder=my_flow_input_encoder,
    # enableTrace=True or omitted entirely still causes the error
))
```

### Possible Solution

Investigate if a specific library version or global configuration variable injects enableTrace unexpectedly.

### Steps to Reproduce

1. Deploy the above code to an AWS Lambda function with the multi_agent_orchestrator library.
2. Attempt to invoke the function:

  - With enableTrace omitted.
  - With enableTrace=True included in BedrockFlowsAgentOptions.

3. Observe the error indicating enableTrace is an unknown parameter, despite the local environment running the same code without issues.

**Comment by brnaba-aws** [comment]
Hi @guisartori88 ,

can you please tell us the version of boto3 you have locally vs the one in lambda?

It might be a boto3 parameter validation issue.

**Comment by brnaba-aws** [comment]
This is still strange because boto3 doesn't make any mention of enableTrace: https://boto3.amazonaws.com/v1/documentation/api/1.35.0/reference/services/bedrock-agent-runtime/client/invoke_flow.html

while the API does: https://docs.aws.amazon.com/bedrock/latest/APIReference/API_agent-runtime_InvokeFlow.html


I'll keep investigate. Thanks

**Comment by guisartori88** [comment]
Exactly! That’s why I was having a hard time. Honestly, it was more luck than following the documentation hahaha

Thank you again for looking into this
